packages/core/src/editor/editor.css

```
@@ -147,23 +187,30 @@
 /* table related: */
-.bn-editor [data-content-type="table"] table {
+/* Scoped to `.bn-root` rather than `.bn-editor`: a table can be rendered
+ outside the contenteditable but still inside the editor's root - most notably
+ in the drag preview, which is mounted into `editor.portalElement` so that it
+ doesn't have to live in the editor's layout to be styled like it. Anything
+ here that's editor chrome rather than the table's own appearance (the
+ `.tableWrapper` padding that makes room for the handles, the drag source
+ highlight) stays scoped to `.bn-editor` above. */
+.bn-root [data-content-type="table"] table {
   width: auto !important;
   word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)